### PR TITLE
Portuguese update

### PR DESCRIPTION
**Pull request by smarquespt**

### CODE REVIEW

**Comment by luis-pereira** [comment]
@smarquespt git push -f your_fork master doesn't work ?

**Comment by smarquespt** [comment]
It works but when I went to my fork on github appears same commits. Those were not squashed and I don´t know why. But since I´m not a developer I don´t know how to fix it.

**Comment by smarquespt** [comment]
Now it works. You can merge it

File: lxqt-config-file-associations/lxqt-config-file-associations.desktop.in [hunk 1]
```diff
@@ -9,3 +9,6 @@ Categories=Settings;DesktopSettings;Qt;LXQt;
 OnlyShowIn=LXDE;LXQt;
 
 #TRANSLATIONS_DIR=translations
+Name[pt]=Associação de ficheiros
```
Comment (luis-pereira) [inline lxqt-config-file-associations/lxqt-config-file-associations.desktop.in]: @smarquespt .desktop.in files are *not* to be edited.
To localize the .desktop file one should create and xxxxxxxxxx_pt.desktop in the translations dir.
Take https://github.com/lxde/lxqt-config/blob/master/lxqt-config-appearance/translations/lxqt-config-appearance_pt.desktop as an example.

I should write some documentation on this.

**Comment by luis-pereira** [comment]
Thanks @smarquespt

**Pull request closed without merging**